clang/include/clang/Driver/Options.td

```diff
@@ -6877,6 +6877,11 @@ defm sycl_esimd_force_stateless_mem : BoolFOption<"sycl-esimd-force-stateless-me
   NegFlag<SetFalse, [], [ClangOption, CLOption], "Do not enforce using "
           "stateless memory accesses.">,
   BothFlags<[], [ClangOption, CLOption, CC1Option], "">>;
+defm sycl_rtc_mode: BoolFOption<"sycl-rtc-mode",
+  LangOpts<"SYCLRTCMode">, DefaultFalse,
+  PosFlag<SetTrue, [], [ClangOption], "Enable">,
```

Comment:
```suggestion
  PosFlag<SetTrue, [], [ClangOption], "Enable runtime compilation mode in SYCL">,
```

Reply:
@srividya-sundaram Thanks for your feedback! After the improvements proposed by @mdtoguchi [here](https://github.com/intel/llvm/pull/15892#discussion_r1822878382), the help text already reads as follows:
```
 -fno-sycl-rtc-mode      Disable RTC mode in SYCL.
 -fsycl-rtc-mode         Enable RTC mode in SYCL.
```

Are you OK if we keep this as is?

Reply:
Thanks for the clarification. I just realized that the text in `Pos/Neg` flags get appended with the text in `BothFlags`.
I was wondering why they just read "Enable" and "Disable" only :)
I am OK with the texts here.